VMT/KCSA/KCSA-CVE-2020-2024.md

```diff
@@ -0,0 +1,68 @@
+announcement-date: 2020-05-28
+
+id: KCSA-CVE-2020-2024
+
+title: Kata Containers improper link resolution vulnerability
+
+description: An improper link resolution vulnerability affects Kata Containers
+versions prior to 1.11.0. Upon container teardown, a malicious guest can trick
+the kata-runtime into unmounting any mount point on the host and all mount
+points underneath it, potentiality resulting in a host DoS.
+
+affected-components:
+
+  - components: `kata-runtime`
+    version: Before v1.11.0
+
+vulnerabilities:
+
+  - cve-id: CVE-2020-2024
+
+reporters:
+
+  - name: Yuval Avrahami
+    affiliation: Yuval Avrahami
```

Comment:
hi @YuvalAvra, which affiliation would you like to be linked to? Generally it can be a company or organization.

Reply:
Palo Alto Networks, thanks